corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -107,8 +107,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
     var showMenu = function (menuResponse) {
         var menuListView = menusUtils.getMenuView(menuResponse);
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
-        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview;
-
+        var queryResponse = menuResponse.queryResponse;
+        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview && queryResponse && queryResponse.displays.length;
```

Comment:
I have mixed feelings about how this gets set. It's a javascript-y thing where `sideBarEnabled` isn't necessarily a boolean. For instance if all else is truth, but `queryResponse.displays.length` is `0`, `sideBarEnabled` is set to `0`. In other circumstances it may be set to `undefined`. This isn't a problem with functionality, and I believe is fine for JS practices.

Reply:
can this be resolved with `queryResponse.displays.length === 0` instead of `queryResponse.displays.length` in that statement?

Reply:
yeah, it would be something like `queryResponse != null && queryResponse.displays.length > 0`. That is how I was doing it at first and I do like it better, personally. There are places where we just go with `arr.length`.